packages/calcite-components/src/components/label/label.e2e.ts

```diff
@@ -145,6 +146,36 @@ describe("calcite-label", () => {
     expect(element).toEqualAttribute("layout", "inline-space-between");
   });
 
+  it(`should emit ${labelClickEvent} when a label is clicked`, async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-label id="do-not-duplicate-me" layout="inline-space-between">
+    Label text
+    <calcite-input></calcite-input>
+    </calcite-label>
+    `);
+
+    const element = await page.find("calcite-label");
+
+    const eventSpy = await element.spyOnEvent(labelClickEvent);
```

Comment:
Can you assert on the selection and unfocused input instead of the internal event (implementation detail)?